src/overlays/actors/ovl_Eff_Dust/z_eff_dust.h

```diff
@@ -6,11 +6,31 @@
 
 struct EffDust;
 
+typedef void (*EffDustActionFunc)(struct EffDust *, GlobalContext *);
+typedef void (*EffDustDrawFunc)(Actor *, GlobalContext *);
```

Comment:
This is the only thing left I think:
```suggestion
typedef void (*EffDustActionFunc)(struct EffDust*, GlobalContext*);
typedef void (*EffDustDrawFunc)(Actor*, GlobalContext*);
```

Reply:
Done!